.codesandbox/templates/next/components/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { createRef, useEffect, useMemo } from 'react';
+import {
+  DOMManager,
+  DOMManagerType,
+  StaticCanvas,
+  StaticDOMManagerType,
+} from 'fabric';
+
+const canvasStyle: React.CSSProperties = {
+  position: 'absolute',
+  left: 0,
+  top: 0,
+  touchAction: 'none',
+  msTouchAction: 'none',
+  userSelect: 'none',
+};
```

Comment:
not sure about the best way to handle this
should it be a default prop on CanvasSlot?